README.md

```diff
@@ -144,6 +141,18 @@ recorder.addEventListener('dataavailable', e => {
 })
 ```
 
+### wav
+As default ,this polyfill saves records to `.wav` files. Compression
+is not very good, but encoding is fast and simple.
+
+### mp3
```

Comment:
The official way to write a name is `MP3` and `WAV`
https://en.wikipedia.org/wiki/MP3

Reply:
done